docs/development_suite/deploy/configure-github-actions.md

```diff
@@ -0,0 +1,126 @@
+---
+id: configure-github-actions
+title:  Configure deployment with GitHub
+sidebar_label: Configure deployment with GitHub
+slug: "/development_suite/deploy/"
+---
+The Mia-Platform Console can be configured to deploy your project through GitHub.
+
+In order to enable your project to be deployed using GitHub, you should edit the configuration of the project through the Console CMS 
+(if you do not have access to it, ask your instance administrator). In particular, the `environments.deploy` object 
+of the configuration should contain the following information:
+
+```json
+{
+  "deploy": {
+    "type": "github-actions",
+    "providerId": "your-github-provider-id"
+  }
+} 
+```
+
+:::info
+if you haven't set up a GitHub provider yet, [see this page](/development_suite/set-up-infrastructure/configure-provider) to learn how to configure it.
+Remember that the Access Token configured to your provider needs to have the `admin` scope of the organization.
+In order for GitHub to send webhook payloads, your server needs to be accessible from the Internet.
```

Comment:
Why it is needed to send a payload? It's needed to trigger the pipeline, I suppose

Reply:
I copied the warnings that Github puts for managing webhooks. 
In general, when Github calls you the target url that you set in the hook, it sends you a payload anyway.
For example this is the payload of the deployment_status event https://docs.github.com/webhooks-and-events/webhooks/webhook-events-and-payloads#deployment_status

Reply:
Instead of "your server", it's better to say "your console instance"

Reply:
```suggestion
In order for GitHub to send webhook payloads, your console instance needs to be accessible from the Internet.
```